commerce-product-type-virtual-order-content-web/src/main/java/com/liferay/commerce/product/type/virtual/order/content/web/internal/display/context/CommerceVirtualOrderItemContentDisplayContext.java

```diff
@@ -304,16 +304,19 @@ public PortletURL getPortletURL() throws PortalException {
 				getLiferayPortletRequest(),
 			getPortletURL(), null, "no-items-were-found");
 
+		long commerceChannelGroupId =
+			_commerceChannelLocalService.getCommerceChannelGroupIdBySiteGroupId(
+				_commerceVirtualOrderItemContentRequestHelper.
+					getScopeGroupId());
+
 		int total =
 			_commerceVirtualOrderItemLocalService.
 				getCommerceVirtualOrderItemsCount(
-					_commerceVirtualOrderItemContentRequestHelper.
-						getScopeGroupId(),
+					commerceChannelGroupId,
 					_commerceVirtualOrderItemContentRequestHelper.getUserId());
```

Comment:
@luca-pellizzon I suppose this is an error and need to pass accountId no userId

Reply:
didn't notice it, but yes it does need accountId